eZ/Publish/Core/FieldType/RelationList/Type.php

```diff
@@ -26,11 +26,17 @@
  */
 class Type extends FieldType
 {
-    /**
-     * @todo Consider to add all 6 selection options
-     */
     const SELECTION_BROWSE = 0;
     const SELECTION_DROPDOWN = 1;
+    /**
+     * @todo following selection methods needs to be implemented in 8.0.
```

Comment:
Actually, some of them where planned to be deprecated. So maybe adjust the comment to cleairfy support _or_ migration instructions should be provided in future version.

But how will backend UI _(content type view/edit and content edit)_ behave with these values? As in both
- 1.0 Platform UI _(given you here add support for it in 6.x)_
- 2.0 Admin UI

Reply:
@andrerom from the PMs discussion in the ticket (https://jira.ez.no/browse/EZP-29924) I understood that the plan is to keep all of them, but maybe I misunderstood something. 

@SylvainGuittard / @bdunogier could you please confirm?

> But how will backend UI (content type view/edit and content edit) behave with these values? As in both

Actually... Either v1 nor v2 doesn't care about these values. It just displays relations in their own templates. This setting is not taken into account.

Reply:
Ok, what I meant to say is with my nitpick:
Let's please not make any promises in doc here that all the methods will be made to be supported in UI. (Som might be deprecated on discovery and migrated to other values for instance, but I did not imply we need to go into details on that).

Simply change comment to be above DROPDOWN to something like: `@todo following selection methods comes from legacy and may be interpreted SELECTION_BROWSE by UI, UI support will be evaluated on a case by case basis.`

Reply:
Thank you, @andrerom. Changed in https://github.com/ezsystems/ezpublish-kernel/pull/2518/commits/8783cb72d46fa40053774b9a415f678a74d84bc5.